test/e2e/scalability/density.go

```diff
@@ -696,7 +697,10 @@ var _ = SIGDescribe("Density", func() {
 			}
 			e2eStartupTime = runDensityTest(dConfig, testPhaseDurations, &scheduleThroughputs)
 			if itArg.runLatencyTest {
-				By("Scheduling additional Pods to measure startup latencies")
+				// Pick latencyPodsIterations so that:
+				// latencyPodsIterations * nodeCount >= MinPodStartupMeasurements.
+				latencyPodsIterations := (MinPodStartupMeasurements + nodeCount - 1) / nodeCount
```

Comment:
This is going to increase time for even our 100-node presubmit. Did you measure how much increase there is?

Reply:
E.g. in https://storage.googleapis.com/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-scalability/16218/build-log.txt latency measuring takes 20 seconds.

Here I'm starting this code 4 more times, so it should increase time by ~1.5 minute (runs usually takes ~40 minutes), so I think it should be fine.

Reply:
Ok, I believe we can live with that.

Reply:
Actually, I just looked at those logs and it seems like the whole batch (creation+watch+deletion) take about ~40s (from 08:17:05 -> 08:17:46). So the increase would actually be ~3 minutes :)

That said, it's still sth we can probably live with.

Reply:
Yes, you are right. I missed pod deletion.